<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-EJSMVPFR" failures="1" tests="7" name="com.GCReddy_v1.TestCases.TC1001_Administrator_loginTest" time="309.016" errors="0" timestamp="2022-10-03T10:24:22 IST" skipped="0">
  <testcase classname="com.GCReddy_v1.TestCases.TC1001_Administrator_loginTest" name="loginWithValidCredentials" time="3.728"/>
  <system-out/>
  <testcase classname="com.GCReddy_v1.TestCases.TC1001_Administrator_loginTest" name="loginWithInvalidCredentials" time="0.540"/>
  <system-out/>
  <testcase classname="com.GCReddy_v1.TestCases.TC1001_Administrator_loginTest" name="loginWithValidUsernameAndInvalidPassword" time="0.458"/>
  <system-out/>
  <testcase classname="com.GCReddy_v1.TestCases.TC1001_Administrator_loginTest" name="loginWithInvalidUsernameAndValidPassword" time="0.452"/>
  <system-out/>
  <testcase classname="com.GCReddy_v1.TestCases.TC1001_Administrator_loginTest" name="maxlogin" time="0.464"/>
  <system-out/>
  <testcase classname="com.GCReddy_v1.TestCases.TC1001_Administrator_loginTest" name="loginAfterMaxLogin" time="301.675">
    <failure message="expected [http://gcreddy.com/project/admin/index.php123] but found [http://gcreddy.com/project/admin/index.php]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [http://gcreddy.com/project/admin/index.php123] but found [http://gcreddy.com/project/admin/index.php]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.GCReddy_v1.TestCases.TC1001_Administrator_loginTest.loginAfterMaxLogin(TC1001_Administrator_loginTest.java:133)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
    </failure>
  </testcase> <!-- loginAfterMaxLogin -->
  <system-out/>
  <testcase classname="com.GCReddy_v1.TestCases.TC1001_Administrator_loginTest" name="redirectingToCustomerTest" time="1.699"/>
  <system-out/>
</testsuite> <!-- com.GCReddy_v1.TestCases.TC1001_Administrator_loginTest -->
